.teamcity/src/main/kotlin/model/PerformanceTestBucketProvider.kt

```diff
@@ -113,7 +113,7 @@ fun splitBucketsByScenarios(scenarios: List<PerformanceScenario>, testProjectToS
         .groupBy({ it.testProject }, { scenario ->
             listOf(testProjectToScenarioDurations, testProjectScenarioDurationsFallback)
                 .mapNotNull { it.getOrDefault(scenario.testProject, emptyList()).firstOrNull { duration -> duration.scenario == scenario.scenario } }
-                .firstOrNull() ?: throw IllegalArgumentException("No durations found for $scenario")
+                .firstOrNull() ?: PerformanceTestDuration(scenario.scenario, 1) // Fallback for new performance tests (don't use 0 to avoid division by zero)
```

Comment:
🤔 Is this necessary? Why?

Reply:
It was an attempt by Bo to make it unnecessary to specify those pesky initial dummy test duration values in `performance-test-durations.json`, but it turned out to be not enough. More changes would be needed and I didn't spend time on figuring out what. But, I left this here, because it definitely doesn't hurt.